### DISCUSSION TITLE
how to extract text or tables from fixed position

### DISCUSSION THREAD

**Discussion by FlyBS** [jsvine/pdfplumber#702] [Ask for help with specific PDFs]
I want to extract text from part fixed position, not all page ranges. Does it support passing two location information, such as (x0,y0),(x1,y1)

![image](https://user-images.githubusercontent.com/19769111/182981474-6ca957f6-4c55-4f53-838a-6d09ac831231.png)

**Comment by jsvine** [comment]
Yes, you can do that with the `.crop(...)` and `.within_bbox(...)` methods. (See the README.md documentation for more details.) try the following:

```python
pdf = pdfplumber.open("my/file.pdf")
page = pdf.pages[0]
print(page.crop((x0, top, x1, bottom)).extract_text())
```

**Reply by FlyBS** [reply]
Yeah, it works, i have a new problem, the x0 or top unit?

**Reply by FlyBS** [reply]
Thanks，i have done.
